src/Connection.php

```diff
@@ -324,14 +323,10 @@ private static function getVersion(): string
 
     private static function lookupVersion(): string
     {
-        if (class_exists(InstalledVersions::class)) {
-            try {
-                return self::$version = InstalledVersions::getPrettyVersion('mongodb/laravel-mongodb');
-            } catch (Throwable) {
-                return self::$version = 'error';
-            }
+        try {
+            return self::$version = InstalledVersions::getPrettyVersion('mongodb/laravel-mongodb');
```

Comment:
```suggestion
            return self::$version = InstalledVersions::getPrettyVersion('mongodb/laravel-mongodb') ?? 'unknown';
```

See: https://github.com/mongodb/mongo-php-library/pull/1262/files#r1531013959

You can also consider factoring out the `lookupVersion()` method entirely.